tutorials/editor/using_the_web_editor.rst

```diff
@@ -0,0 +1,120 @@
+- No GDNative support.
+- No way to download the project files as a ZIP archive.
+- No project exporting.
+- The editor won't warn you when closing the tab with unsaved changes.
+- No lightmap baking support. You can still use existing lightmaps if they were
+  baked with a native version of the Godot editor.
+
+The following features are unlikely to be supported due to inherent limitations
+of the Web platform:
+
+- No support for external script editors.
+- No support for Android one-click deploy.
+
+.. seealso::
+
+    See the
+    `list of open issues on GitHub related to the web editor <https://github.com/godotengine/godot/issues?q=is%3Aopen+is%3Aissue+label%3Aplatform%3Ahtml5+label%3Atopic%3Aeditor>`__ for a list of known bugs.
+
+Importing a project
+-------------------
+
+To import an existing project, the current process is as follows:
+
+- Specify a ZIP file to preload on the HTML5 filesystem using the
+  **Preload project ZIP** input.
+- Run the editor by clicking **Start Godot editor**.
+  The Godot project manager should appear after 10-20 seconds.
+  On slower machines or connections, loading may take up to a minute.
+- Click the **Import** button on the right, click **Browse** in the dialog that appears
+  and choose ``preload.zip``.
+- Click the **Browse** button next to the Project Installation Path. Double-click
+  the ``projects`` folder then click **Create Folder** in the top-right corner of
+  the file dialog.
+- Specify a name for the project folder (it doesn't have to match the ZIP archive's name).
+  Click **OK**.
+- Now that the folder has been created, click **Select Current Folder**.
+  If everything went right, the Project Installation Path should look like this:
+  ``/home/web_user/projects/<project folder>``.
+- Click **Import** and the project will open in the editor.
+
+.. attention::
+
+    It's important to place the project folder in ``/home/web_user/projects``.
+    If your project folder is placed outside ``/home/web_user/projects``, you will
+    lose your project when closing the editor!
+
+This import process will be streamlined in the future.
+
+Editing and running a project
+-----------------------------
+
+Unlike the native version of Godot, the web editor is constrained to a single
+window. Therefore, it cannot open a new window when running the project.
+Instead, when you run the project by clicking the Run button or pressing
+:kbd:`F5`, it will appear to "replace" the editor window.
+
+The web editor offers an alternative way to deal with the editor and game
+windows (which are now "tabs"). You can switch between the **Editor** and
+**Game** tabs using the buttons on the top. You can also close the running game
+or editor by clicking the **×** button next to those tabs.
+
+Where are my project files?
+---------------------------
+
+Due to browser security limitations, the editor will save the project files to
+the browser's IndexedDB storage. This storage isn't accessible as a regular folder
```

Comment:
Does the editor actually use IndexedDB for storage? I don't remember exactly; I may be incorrect there.

Reply:
yes, we do use IndexedDB,  the first prototype also had FileSystemAPI support but was pretty messy with an external library so I decided to drop it, since IndexedDB seems to work fine.